src/main/java/pages/CatalogPage.java

```diff
@@ -13,6 +16,7 @@ public CatalogPage() {
 
     public ElectronicsPage clickOnElectronicsPageLink() {
         electronicsLink.click();
+        logger.debug("Clicked successfully on the button : " + electronicsLink.getText());
```

Comment:
I think it's better not to use electronicsLink.getText() as it's an extra interaction with the element, it might cause unexpected test failures (if the element is not interactable or smth), for log messages try to use pour strings without interacting with elements